docs/architecture/README.md

```diff
@@ -52,6 +52,8 @@ numbering our ADRs from 100 onwards.
 
 ### Implemented
 
+- [ADR-108: E2E tests for CometBFT's behaviour in respect to ABCI 1.0.](./adr-108-e2e-abci++.md)
```

Comment:
Has this ADR actually been implemented yet?

Reply:
It is implemented here - https://github.com/cometbft/cometbft/pull/930. However, it still needs to be reviewed.

Reply:
> we also need a tracking issue for its implementation

It's #353, I just edited its DoD to refer to the actual PRs